src/Game.cpp

```diff
@@ -75,9 +233,29 @@ void check_sku_version() {
 // mc2: 0x0053B9B0
 // Likely related to assets_p: /tune/hud/ntsc/
 void sub_53B9B0() {
-    if (sub_612E10("pc_480")) glo_66315C = 7; // 480 x 640
-    else if (sub_612E10("pc_720")) glo_66315C = 8; // 720 x 480
-    else glo_66315C = 0;
+    if (sub_612E10("pc_480")) glo_66315C = 7; // 480 x 720
+    else if (sub_612E10("pc_720")) glo_66315C = 8; // 720 x 1280
+    else glo_66315C = 0; // 480 x 640
+}
+
+void sub_53B9F0() {
+    int resolution[3][2] = {
+        {640, 480},
+        {720, 480},
+        {1280,720}
+    };
```

Comment:
I'm probably going to remove this array and just make the function calls directly. That, or I'm going to make the table `constexpr` to allow for more compiler optimization.